torch/csrc/DynamicTypes.cpp

```diff
@@ -69,7 +69,11 @@ at::Type* get_type(const std::string& name, bool is_cuda, bool is_sparse) {
 
 PyTypeObject* getPyTypeObject(const at::Storage& storage)
 {
-  auto it = attype_to_py_storage_type.find(&storage.pImpl()->type());
+  auto attype = get_type(
+      at::toString((storage.pImpl())->scalar_type()),
```

Comment:
This looks really weird. Why are you calling `at::toString` here?

Reply:
Because get_type needs a name to look up the correct type.